### ISSUE TITLE
Codecept : Cannot find module '@babel/runtime/helpers/interopRequireDefault'

### ISSUE THREAD

**Issue by soufianetomase** [Codeception/CodeceptJS#1983]
Hello,

I try to simply run the tutorial "https://codecept.io/quickstart"

```
$ npx codeceptjs init
npx: 1 installé(s) en 3.08s
Path must be a string. Received undefined
.....\node_modules\codeceptjs\bin\codecept.js
Cannot find module '@babel/runtime/helpers/interopRequireDefault'
```

i think one or more dependencies have been updated without ensuring compatibility.

Even if I use an old version of codecept I have the same error.

when i add the dependency ''@babel/runtime' i have a new error :

```
$ npx codeceptjs init
npx: 1 installé(s) en 3.03s
Path must be a string. Received undefined
.....\node_modules\codeceptjs\bin\codecept.js
Cannot find module './utils'
```

**Comment by dustinlessard-wf** [comment]
We're hitting the same issues.

**Comment by zarashima** [comment]
For that error, I solve it using the suggested solution here: https://github.com/Codeception/CodeceptJS/issues/1976
`npm install parse-function@5.2.11`

**Comment by dustinlessard-wf** [comment]
@zarashima , that did not solve our issue.  We continue to hit '@babel/runtime/helpers/interopRequireDefault'

**Comment by dustinlessard-wf** [comment]
We ended up needing to blow away our node_modules directory, then got it work!

**Comment by pablopaul** [comment]
This is fixed with the 2.3.5 release, can you confirm and close the issue?

**Comment by soufianetomase** [comment]
done !
